js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ExchangeNotAvailable, OnMaintenance, ArgumentsRequired, BadRequest, AccountSuspended, InvalidAddress, PermissionDenied, DDoSProtection, InsufficientFunds, InvalidNonce, CancelPending, InvalidOrder, OrderNotFound, AuthenticationError, RequestTimeout, NotSupported, BadSymbol, RateLimitExceeded } = require ('./base/errors');
+const { TICK_SIZE } = require ('./base/functions/number');
+const Precise = require ('./base/Precise');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class bitget3 extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'bitget3',
+            'name': 'Bitget',
+            'countries': [ 'SG' ],
+            'version': 'v3',
+            'rateLimit': 1000, // up to 3000 requests per 5 minutes ≈ 600 requests per minute ≈ 10 requests per second ≈ 100 ms
```

Comment:
We have different rates for different endpoints and we're not differentiating them here. Consider adding weights to the endpoints

![image](https://user-images.githubusercontent.com/43336371/152368815-97835281-df7c-40d5-a34c-35b657433223.png)
![image](https://user-images.githubusercontent.com/43336371/152368881-cca9ef0e-a0f3-4359-9bca-49676d8d71ba.png)
![image](https://user-images.githubusercontent.com/43336371/152368932-2caff357-524f-46c5-b749-11e6c6a16cd3.png)

Reply:
Fixed in PR #11907